GitUI/UserControls/FilterToolBar.cs

```diff
@@ -362,6 +356,21 @@ private void revisionGridFilter_FilterChanged(object? sender, FilterChangedEvent
                 }
             }
 
+            // Add to dropdown and settings, unless already included
+            string filter = tstxtRevisionFilter.Text;
+            if (!string.IsNullOrWhiteSpace(filter) && (tstxtRevisionFilter.Items.Count == 0 || filter != (string)tstxtRevisionFilter.Items[0]))
+            {
+                if (tstxtRevisionFilter.Items.Contains(filter))
+                {
+                    tstxtRevisionFilter.Items.Remove(filter);
+                }
+
+                tstxtRevisionFilter.Items.Insert(0, filter);
```

Comment:
Should we at least trim the text before comparing/storing?

Reply:
ok

Reply:
The combobox text is trimmed, so all uses are trimmed